module2/web/src/main/resources/application.yml

```diff
@@ -0,0 +1,14 @@
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: jdbc:postgresql://localhost:5432/gift_certificates_system
+    username: postgres
+    password: root
```

Comment:
It is better not to store username and password like that. It is sensitive information. You can use ${USERNAME} here and pass values via env vars or command vars

Reply:
fixed